cmd/agent/app/agent.go

```diff
@@ -138,6 +139,16 @@ func run(ctx context.Context, karmadaConfig karmadactl.KarmadaConfig, opts *opti
 	if err != nil {
 		return err
 	}
+
+	ok, err := util.IsClusterIDUnique(karmadaClient, id)
+	if err != nil {
+		return err
+	}
+
+	if !ok {
+		return fmt.Errorf("cluster(id):%s already exist", id)
```

Comment:
> error: failed to create cluster(member4) object. error: cluster(id:fc1c4701-34c9-4a4b-afc2-296ecefe0c2d) already exist

This log doesn't clear enough for users to understand what happened.
The log should be `The same cluster has been registered with name xxxx`